### DISCUSSION TITLE
Plan output contains symbols/ ascii chars, is it expected ?

### DISCUSSION THREAD

**Discussion by DanyC97** [dflook/terraform-github-actions#269] [General]
Hi, 

i thought i should start a convo instead of an issue, if it turns out being valid issue will move it there.

Using tf-plan action and the `cloud` backend type where my workspace is being configured as `remote`  (in the execution mode section), while running a plan and the output is being pushed as a comment i get

```
Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/foo-org/tfc-mgmt/runs/run-mDwwA5RTAopqLAZJ

Waiting for the plan to start...

Terraform v1.4.5
on linux_amd64
Initializing plugins and modules...
�[0m�[1mdata.tfe_team.teams["Release Mgmt"]: Reading...�[0m�[0m
�[0m�[1mtfe_workspace.workspaces["terraform-google-cloudnat"]: Refreshing state... [id=ws-yGr1smKWAavJvD7T]�[0m

...


�[0m�[1m�[32mNo changes.�[0m�[1m Your infrastructure matches the configuration.�[0m

�[0mTerraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.
```

Is this expected or am i doing s'thing wrong?

Please note the output seen in TFC UI match the one seen in the GH action job log.

On a different tangent note, when running in `cloud` mode is the expectation for the plan output to be just the url as a comment or what i see above is the right behaviour ?

**Comment by DanyC97** [comment]
A minor update - when i change the execution mode to `local` then the gh comment output is shown correctly .. could this be another edge case 🤔 
![image](https://github.com/dflook/terraform-github-actions/assets/7383422/220450cd-7f1a-4aa4-bca7-49bf6d5b102e)

**Reply by dflook** [reply]
Hi, I think this can be raised as an issue.

The weird characters are terraform adding text colouring, which doesn't work in GitHub comments - which is why we turn it off. It seems like when in 'remote' planning mode, TFC isn't respecting the '-no-color' flag.

What you should be seeing is the full plan in the comment, essentially as if you had run it in local mode (except for the Running in Terraform Cloud preamble)

**Comment by DanyC97** [comment]
thank you @dflook , will close this discussion as the issue been raised.

Appreciate your quick response 🙌
